pkg/storage/stores/shipper/compactor/deletion/delete_request.go

```diff
@@ -97,13 +97,30 @@ func (d *DeleteRequest) IsDeleted(entry retention.ChunkEntry) (bool, []retention
 		return false, nil
 	}
 
+	level.Debug(util_log.Logger).Log(
+		"msg", "starting filter function",
+		"delete_request_id", d.RequestID,
```

Comment:
should we also use camel-case here ? I see that for `userID` we use camel-case

Reply:
Due to a previous comment `userID` is now changed to `user`. Regarding camel-case or not, we seem to be somewhat inconsistent. There are a lot of examples of each style in the code.

Reply:
No, we generally use snake_case in our log keys

Reply:
ok, great ;)